src/reducers/cartReducer.js

```diff
@@ -4,7 +4,10 @@ export const cartReducer = (
 ) => {
 	switch (type) {
 		case "ADD":
+			
+
 			cart = [...payload];
```

Comment:
abstract out the spread of payload on top of switch condition

Reply:
Done